google-cloud-tools-plugin/testSrc/com/google/cloud/tools/intellij/PropertiesTest.java

```diff
@@ -35,9 +35,7 @@ public void testSpace() throws IOException {
       in = new FileInputStream(f);
       Properties p = new Properties();
       p.load(new InputStreamReader(in, "UTF-8"));
-      Assert.assertEquals(
-          "Do you want to checkout branch {0} and restore the saved stash?",
-          p.getProperty("clouddebug.restorestash"));
+      Assert.assertEquals("Google Cloud Tools", p.getProperty("cloud.tools.group.menu.text"));
```

Comment:
This test looks pretty useless in general, how about I'll file a separate PR to remove it?

Reply:
knock yourself out. :)